<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>抓包器</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Interface:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox"/>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Protocol:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_2">
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>IPV4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ARP</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ICMP</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ETHERNET</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>UDP</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>IP:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>MAC:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_2"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QTableWidget" name="tableWidget">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>120</number>
       </attribute>
       <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderStretchLastSection">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Time</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Source</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Destination</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Protocol</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Length</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Info</string>
        </property>
       </column>
      </widget>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QTreeWidget" name="treeWidget">
        <property name="minimumSize">
         <size>
          <width>440</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Liberation Mono</family>
          <kerning>false</kerning>
         </font>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QPlainTextEdit" name="plainTextEdit_3">
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Liberation Mono</family>
             <kerning>false</kerning>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(239, 239, 239);
color: rgb(83, 83, 83);
</string>
           </property>
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="plainText">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="plainTextEdit">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Liberation Mono</family>
             <pointsize>9</pointsize>
             <kerning>false</kerning>
            </font>
           </property>
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="plainText">
            <string/>
           </property>
           <property name="backgroundVisible">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="plainTextEdit_2">
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Liberation Mono</family>
             <kerning>false</kerning>
            </font>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="plainText">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
        <zorder>plainTextEdit</zorder>
        <zorder>plainTextEdit_2</zorder>
        <zorder>plainTextEdit_3</zorder>
       </widget>
      </widget>
      <widget class="QWidget" name="formLayoutWidget">
       <layout class="QFormLayout" name="formLayout"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionstart"/>
   <addaction name="scrollBottom"/>
  </widget>
  <action name="actionstart">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="go-next">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>开始抓包</string>
   </property>
  </action>
  <action name="scrollBottom">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="go-bottom">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>下拉到底</string>
   </property>
   <property name="toolTip">
    <string>下拉到底</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
